### ISSUE TITLE
Locale Selector: Resize Text - Tab/shift tab navigation in 200% zoom text (Location and languages page)

### ISSUE THREAD

**Issue by chetankan** [carbon-design-system/ibm-dotcom-library#1764]
Issue :- Doing Tab/shift tab navigation focus is going to the element but focused element is not visible to the user.

Expected :- The page should automatically scrolled up/down once the interactive element receives the tab focus so that the element which receives the tab focus should visible to the user clearly.

**Comment by RaphaelAmadeu** [comment]
@chetankan 
I've recorded this video testing the issue, and it seems to be looking fine
https://ibm.ent.box.com/file/649714585309

Can you tell me if it's still occurring to you?

Thanks!

**Comment by kennylam** [comment]
@chetankan @Praveen-sr @RobertaJHahn 
https://github.com/carbon-design-system/ibm-dotcom-library/pull/2174
